superset/db_engine_specs/snowflake.py

```diff
@@ -14,12 +14,22 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import re
 from datetime import datetime
-from typing import Optional
+from typing import List, Optional, TYPE_CHECKING
 from urllib import parse
 
+import pandas as pd
+
+from superset import cache
 from superset.db_engine_specs.postgres import PostgresBaseEngineSpec
 
+if TYPE_CHECKING:
+    # prevent circular imports
+    from superset.models.core import Database  # pylint: disable=unused-import
+
+func_regex = re.compile(r"^((?<!SYSTEM$)(\w))+$")
```

Comment:
Nit. `func_regex` -> `FUNC_REGEX` as it's globally defined. Note if configured correctly `pylint` should error on this.

Reply:
Fixed.